src/main/java/org/isf/sms/model/Sms.java

```diff
@@ -1,23 +1,24 @@
 /*
- * Open Hospital (www.open-hospital.org)
- * Copyright © 2006-2020 Informatici Senza Frontiere (info@informaticisenzafrontiere.org)
```

Comment:
Why change the default license block that is generated by a maven target?  Please rollback.

Reply:
restored